src/app/common/resource-network/resource-network.component.html

```diff
@@ -13,5 +13,9 @@
       <div id="indicator-icon" class="legend-icon"></div>
       <div class="legend-text">Indicator</div>
     </div>
+    <div class="legend-icon-container">
+      <div id="recipe-icon" class="legend-icon"></div>
+      <div class="legend-text">Guideline</div>
+    </div>
```

Comment:
Guideline seems to be appearing at the bottom of the legend. Is that what you intened?

Reply:
there wasn't a legend for recipe/guideline type resource. I added it. the order does not matter